src/migrations/00013-add-device-types.sql

```diff
@@ -1,29 +1,28 @@
 -- create device types table
 
-CREATE TABLE IF NOT EXISTS "device type" (
+CREATE TABLE IF NOT EXISTS "device type table" (
```

Comment:
Ouch. How did this slip though? Can you please make this a separate PR that is merged as soon as possible? The API is completely broken now.

Reply:
@dfunckt  
Here is the PR that fixes this: https://github.com/balena-io/open-balena-api/pull/179

I also realized that we have two 00014 migrations, one of them should become 00015.